tests/test_project_renaming_compatibility.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import patch
+from uuid import uuid4
+
+import pytest
+
+from pasqal_cloud import SDK
+from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
+
+
+"""
+Ticket (#622), Python file to be entirely removed
+"""
+
+
+@patch("pasqal_cloud.client.Auth0TokenProvider", FakeAuth0AuthenticationSuccess)
+class TestSDKAndClientInit:
+    group_id = "random_group_id"
+    project_id = "random_project_id"
+    username = "random_username"
+    password = "random_password"
+
+    def test_client_group_argument_converted_to_project(self):
+        sdk = SDK(group_id=self.group_id, username=self.username, password=self.password)
+        assert sdk._client.project_id == self.group_id
+
+    def test_client_project_and_group_given(self):
```

Comment:
Looks like this test is not doing what the name implies

Reply:
changed